src/PlaneWaveBasis.jl

```diff
@@ -39,10 +39,9 @@ Normalization conventions:
 
 `ifft` and `fft` convert between these representations.
 """
-struct PlaneWaveBasis{T, VT, T_G_vectors, T_r_vectors, T_kpt_G_vecs} <: AbstractBasis{
-    T
-} where {VT <: Real, T_G_vectors <: AbstractArray{Vec3{Int}}, T_r_vectors <: AbstractArray,
-         T_kpt_G_vecs <: AbstractVector{Vec3{Int}}}
+struct PlaneWaveBasis{T, VT, T_G_vectors, T_r_vectors, T_kpt_G_vecs} <: AbstractBasis{T} where
+    {VT <: Real, T_G_vectors <: AbstractArray{Vec3{Int}},
+    T_r_vectors <: AbstractArray{Vec3}, T_kpt_G_vecs <: AbstractVector{Vec3{Int}}}
```

Comment:
Can't this be AbstractArray{Vec3{T}, 3}?
Sorry for the bother, but it does help readers to know what is what
(also skip lines)

Reply:
Getting the `T` in is tricky, but we can try the 3

Reply:
no this seems to work